.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Why is this necessary?

Reply:
@domenic It is not necessary, but my thinking was that it makes things more "consistent" in behavior. 

Let's presume your install the packages locally, without it, `npm` will still generate the `package-lock.json` file that, for example, will lock package `P` to version `x.y.1`. When version `x.y.2` is released, locally (unless  the `package-lock.json` is removed or modified) you will still get version `x.y.1`, while on Travis (because with every build you start from scratch and a new `package-lock.json` is created), you will get `x.y.2`. 

---

Let me know if you want me to remove it.

Reply:
I guess that makes sense; thank you!